### PR TITLE
[FIX] hr_holidays: Timeoff domain in view

### PR DESCRIPTION
**Pull request by fdamhaut**
Timeoff is apply in the view to take effect on every view type
instead of only the calendar.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by Williambraecky — APPROVED** [review]
LGTM Thanks :) and tested seems to work correctly.

@kba-odoo

File: addons/hr_holidays/views/hr_leave_views.xml [hunk 1]
```diff
@@ -648,7 +648,7 @@
             'search_default_managed_people': 1,
             'hide_employee_name': 1}
         </field>
-        <field name="domain">[]</field>
+        <field name="domain">[('employee_id.company_id', 'in', company_ids)]</field>
```
Comment (kba-odoo) [inline addons/hr_holidays/views/hr_leave_views.xml:651]: Shouldn't it be `allowed_company_ids`? :thinking:

**Comment by fdamhaut** [comment]
@kba-odoo changed(and tested) according to your suggestion :D

**Comment by kba-odoo** [comment]
robodoo r+

**Pull request closed without merging**